liquid-prelude/src/Language/Haskell/Liquid/Bag.hs

```diff
@@ -17,6 +17,7 @@ import qualified Data.Map      as M
 @-}
 
 
+{-@ type Bag a = Data.Map.Map a Nat @-}
```

Comment:
why not name the LH alias `NatBag`?

Reply:
The alias could be renamed. Then `NatBag String` is a bag of `String`s (not Nats). Will that be confusing?

Reply:
I see... How about `RBag` from Refined Bag?
Just trying to prevent the requirement to have to give refined signatures to each user of `Bag`.